CHANGELOG-filefeeds.md

```diff
@@ -7,6 +7,13 @@ This log is intended to keep track of package changes, including
 but not limited to API changes and file location changes. Minor behavioral
 changes may not be included if they are not expected to break existing code.
 
+## 0.5.0 (2024-10-03)
+
+- Support `session_token` as an init and launch parameter. When the session
+  token for an existing, valid session is passed in along with a corresponding,
+  jwt, the specified session will be resumed. Session token will also be stored 
+  in localstorage upon init success and polled for when none is provided.
```

Comment:
@brianbrianschema, could you please split the in-built resume session (even for newly-created session) in a separate change item?

And, considering that the resume-session behavior is the customer-facing change, we could skip sharing the details about the localstorage being used for it and only focus on the library behavior from the outside?